core/src/main/java/bisq/core/dao/node/full/RpcService.java

```diff
@@ -350,9 +352,14 @@ private static RawTx getTxFromRawTransaction(RawDtoTransaction rawDtoTx,
                                     }
                                 }
                             }
-                            // We don't support raw MS which are the only case where scriptPubKey.getAddresses()>1
-                            String address = scriptPubKey.getAddresses() != null &&
```

Comment:
Is the addresses field in `DtoPubKeyScript` still valid/supported?

Reply:
AFAIK the addresses field in `DtoPubKeyScript` is populated when bitcoind is older than v22; `null` if >= v22.

I think it can be removed if that is what you prefer, I will test it out with an older `bitcoind` to make sure that is ok.  Noticed that removing it from code also cascades to DAO `PubKeyScript` and `JsonScriptPubKey` so will require some more testing.

Reply:
Better leave it, just to be sure that it does not cause an exception, but seems not.

Reply:
Btw. I have not tested with mainnet, would be good to get some full node setup with mainnet tested as well

Reply:
@jmacxx Have you tested it with Mainnet as well?

Reply:
I am working on mainnet tests, it is going slow due to syncing fullnodes and repeatedly running out of disk space.  Will update here when tests complete.

Reply:
The test failed, DAO hashes no longer matched with the seednodes.  Investigation showed that the `addresses` field makes its way into the DAO state records and thus when it is null, affects the DAO state hash.  The way to solve this is to generate the addresses field if necessary at the point we get DtoPubKeyScript from bitcoind.